app/bundles/CoreBundle/Assets/js/7.charts.js

```diff
@@ -61,10 +66,41 @@ Mautic.renderLineChart = function(canvas) {
         options: {
             lineTension : 0.2,
             borderWidth: 1,
+            tooltips: {
+                mode: 'index',
+                intersect: false
+            },
             scales: {
+                xAxes: [{
+                    gridLines: {
+                        display: false
+                    }
+                }],
                 yAxes: [{
+                    afterBuildTicks: function(scale) {
+                        scale.ticks = [];
+                        scale.ticks.push(scale.min);
+                        scale.ticks.push((scale.max - scale.min) / 2);
+                        scale.ticks.push(scale.max);
+                    },
+                    gridLines: {
+                        drawBorder: false,
+                    },
                     ticks: {
-                        beginAtZero: true
+                        beginAtZero: true,
+                        callback: function(value, index, values) {
+                            if (index === 0 || index === values.length - 1) {
+                                return value;
+                            }
+                            if (value === 0.5) {
+                                return '';
+                            }
```

Comment:
This fixed the `0.5` value but not higher numbers like `1.5`
![Screenshot 2024-04-17 at 17 28 07](https://github.com/mautic/mautic/assets/1235442/f7e3dff4-79e8-41da-a16a-c48994af020b)
How about
```suggestion
                            if (/^\d+\.5$/.test(value.toString())) {
                                return '';
                            }
```

Reply:
Way better solution, thanks!